src/components/app.js

```diff
@@ -236,7 +237,8 @@ const App = () => {
                                 <GridActionsLogoDark />
                             )
                         }
-                        onParametersClick={() => showParametersClicked()}
+                        //To uncomment when parameters are added to gridactions
+                        // onParametersClick={() => showParametersClicked()}
```

Comment:
Maybe, the 'Parameters' item should be disabled, because there are no parameters yet.
And, we cannot change theme anymore

Reply:
i agree with Franck we should disable parameters item.

Reply:
That's the behavior when we don't give the call back. As the callback is undefined, parameters button will be disabled in commons ui.

Reply:
Commented code has been removed.